File: client/src/main/kotlin/org/bitlap/jdbc/BitlapMetaDataResultSet.kt
/* Copyright (c) 2022 bitlap.org */
package org.bitlap.jdbc

/**
 *
 * @author 梦境迷离
 * @since 2021/8/23
 * @version 1.0
 */
abstract class BitlapMetaDataResultSet<M>(
    override var columnNames: MutableList<String> = ArrayList(),
    override var columnTypes: MutableList<String> = ArrayList(),
    protected var data: List<M> = mutableListOf()
) : BitlapBaseResultSet() {

    override fun close() {
    }
}

File: client/src/main/scala/org/bitlap/jdbc/BitlapMetaDataResultSet.scala
/* Copyright (c) 2022 bitlap.org */
package org.bitlap.jdbc

/**
 * @author 梦境迷离
 * @since 2021/8/23
 * @version 1.0
 */
abstract class BitlapMetaDataResultSet[M](
  override val columnNames: List[String] = List(),
  override val columnTypes: List[String] = List(),
  protected var data: List[M] = List()
) extends BitlapBaseResultSet {

  override def close(): Unit = {}
}
